StRoot/St_geant_Maker/GtHash.cxx

```diff
@@ -19,26 +19,26 @@ GtCradle(int n);
 GtCradle::GtCradle(int n)
 {
  fNWords = n;
- fArray  = new ULong_t[fNWords]; 
+ fArray  = new UInt_t[fNWords]; 
  fPointer=0;
 }
  
 Bool_t GtCradle::IsEqual(const TObject* obj) const
 { 
   GtCradle *He = (GtCradle*)obj;
   if (fNWords != He->fNWords) return 0; 
-  ULong_t *me  = fArray;
-  ULong_t *he  = He->fArray;
+  UInt_t *me  = fArray;
+  UInt_t *he  = He->fArray;
   if (me[0]   != he[0]) return 0; 
   for (int i=1; i<fNWords; i++) if (me[i]!=he[i]) return 0;
   return 1;
 }
 ULong_t GtCradle::Hash() const
 { 
-  ULong_t *me = (ULong_t *)fArray;
-  ULong_t ret = fNWords;
+  UInt_t *me = (UInt_t *)fArray;
+  UInt_t ret = fNWords;
 //VP  for (int i=0; i<fNWords; i++) ret ^= me[i];
-  ret = TMath::Hash(me,fNWords*sizeof(ULong_t));
+  ret = TMath::Hash(me,fNWords*sizeof(UInt_t));
```

Comment:
The documentation for `TMath::Hash()` indicates that this has been moved to TString::Hash() and the function in TMath is only retained for backward compatibility. I checked ROOT 6.32, and `TMath::Hash()` is still there, with the same message. Nevertheless, maybe we want to switch to TString just in case?
```suggestion
  ret = TString::Hash(me,fNWords*sizeof(UInt_t));
```